CHANGELOG.unreleased.md

```diff
@@ -16,6 +16,7 @@
 - [Host/Advanced] Fix _Hyper Threading_ not correctly recognized if _Smartctl_ plugin returned an error [Forum#8675](https://xcp-ng.org/forum/topic/8675/ht-smt-detection-in-8-3-not-fully-working) (PR [#7525](https://github.com/vatesfr/xen-orchestra/pull/7525))
 - [VMWare/Migration] Use NFS datastore for import from XO5 (PR [#7530](https://github.com/vatesfr/xen-orchestra/pull/7530))
 - [VMWare/Migration] fix `Can't import delta of a running VM without its parent vdi``when importing snapshotless VM   (PR [#7530](https://github.com/vatesfr/xen-orchestra/pull/7530))
+- [VMWare/Migration] ignore VM without storage (PR [#7530](https://github.com/vatesfr/xen-orchestra/pull/7530))
```

Comment:
```
- [VMWare/Migration] Don't fail all VMs if one does not have any disks (PR [#7530](https://github.com/vatesfr/xen-orchestra/pull/7530))
```

Reply:
git absorb is magic

Reply:
Yes, if only GitHub could support autosquashing PRs.